blockchain/package.json

```diff
@@ -14,7 +14,11 @@
   "license": "ISC",
   "description": "",
   "devDependencies": {
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.11",
```

Comment:
@Ratawine7 This is a peer dependency of **@nomicfoundation/hardhat-toolbox** so you do not have to install it alone.